chain-crypto/src/ec/mod.rs

```diff
@@ -1,9 +1,14 @@
-mod babystep;
+//! Module defining the Group Elements and Scalar structures in one primer order group (over sec2
+//! curves), or the other (ristretto255).
+#[macro_use]
+mod macros;
 #[cfg(not(feature = "ristretto255"))]
```

Comment:
I don't like the idea of an absence of feature bringing in a different backend implicitly.

Maybe there should be backend selection features `ec-p256k1` and `ec-ristretto255`, where the first one is listed among default features?

Reply:
I'm not sure I understood this. Do you mean that we should do `#[cfg(feature = "p256k1")]` instead of `#[cfg(not(feature = "ristretto255"))]`?

Reply:
there are already different features for the two backends with p256k1 being among default ones. AFAIU The reason for `#[cfg(not(...))` is to guarantee that one and only one backend is selected at a time